src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * 
+ * @author Naren
+ * Association class for keeping track of changes to tags
```

Comment:
A more precise description is `Represents an adding or removing a Tag to/from a Person`.